pkg/apis/aws/v1alpha1/shared_types.go

```diff
@@ -88,8 +88,11 @@ var ErrCreateEC2Instance = errors.New("EC2CreationTimeout")
 // ErrFailedAWSTypecast indicates that there was a failure while typecasting to aws error
 var ErrFailedAWSTypecast = errors.New("FailedToTypecastAWSError")
 
-// ErrInvalidOUMap indicates that the OU ConfigMap is not valid
-var ErrInvalidOUMap = errors.New("OUConfigMapInvalid")
+// ErrMissingDefaultConfigMap indicates that the expected default confimap was not found
+var ErrMissingDefaultConfigMap = errors.New("MissingDefaultConfigMap")
+
+// ErrInvalidConfigMap indicates that the ConfigMap has invalid fields
+var ErrInvalidConfigMap = errors.New("OUConfigMapInvalid")
```

Comment:
Is this different than the account limit being invalid?  Any reason not to change to `ConfigMapInvalid`?

Reply:
@fahlmant I missed updating the error name thanks for catching that! I use this error to encompass all cases of missing or bad field values. Are you saying we should have errors specific to each field that is missing/invalid?

Reply:
No I'm saying we should just have one `ConfigMapInvalid`

Reply:
Does my latest commit fix that concern?

Reply:
👍